Microsoft.Toolkit.Uwp.UI.Controls.Layout/VisualStudioToolsManifest.xml

```diff
@@ -0,0 +1,6 @@
+<FileList>
+  <File Reference="Microsoft.Toolkit.Uwp.UI.Controls.Layout.dll">
+    <ToolboxItems VSCategory="Windows Community Toolkit" BlendCategory="Windows Community Toolkit">
```

Comment:
Is this the new format or the old way?

Reply:
This was copied from the controls project format